src/DssSpell.sol

```diff
@@ -59,188 +45,15 @@ contract DssSpellAction is DssAction {
-
-        // 0xRoot - 8.34 MKR - 0xC74392777443a11Dc26Ce8A3D934370514F38A91
-        MKR.transfer(ROOT, 8.34 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
-
-        // ---------- Aave Revenue Share ----------
-        // Forum: https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-3-q1-2024/24014
-
-        // Transfer 238,339 DAI to 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c
-        DssExecLib.sendPaymentFromSurplusBuffer(AAVE_V3_TREASURY, 238_339);
-
-        // ---------- Whitelist new address in the RWA015-A output conduit ----------
-        // Forum: https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084
-
-        // Call kiss on RWA015_A_OUTPUT_CONDUIT with address 0x6759610547a36E9597Ef452aa0B9cace91291a2f
-        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).kiss(RWA015_A_CUSTODY_2);
-
-        // ---------- Push USDP out of input conduit ----------
-        // Forum: https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084
-
-        // Raise PSM-PAX-A DC to 100,000 DAI
-        DssExecLib.setIlkDebtCeiling("PSM-PAX-A", 100_000);
-
-        // Call push() on MCD_PSM_PAX_A_INPUT_CONDUIT_JAR (use push(uint256 amt)) to push 84,211.27 USDP
-        InputConduitJarLike(MCD_PSM_PAX_A_INPUT_CONDUIT_JAR).push(84_211.27 ether); // Note: `ether` is only a keyword helper
-
-        // Call void() on MCD_PSM_PAX_A_JAR
-        JarLike(MCD_PSM_PAX_A_JAR).void();
-
-        // Set PSM-PAX-A DC to 0 DAI
-        DssExecLib.setIlkDebtCeiling("PSM-PAX-A", 0);
-
-        // ---------- Spark Proxy Spell ----------
-        // Forum: https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033
-        // Poll: https://vote.makerdao.com/polling/QmZND8WW
-        // Poll: https://vote.makerdao.com/polling/QmcRdMyA
-        // Poll: https://vote.makerdao.com/polling/QmSh8gyC
-        // Poll: https://vote.makerdao.com/polling/QmfGV2vt
-        // Poll: https://vote.makerdao.com/polling/QmSYZSCQ
-        // Poll: https://vote.makerdao.com/polling/QmUhT32b
-        // Poll: https://vote.makerdao.com/polling/QmVsKsGa
-        // Forum: https://forum.makerdao.com/t/sparklend-external-security-access-multisig-for-freezer-mom/24070
-        // Poll: https://vote.makerdao.com/polling/QmVXriiT
-        // Forum: http://forum.makerdao.com/t/stability-scope-parameter-changes-12-under-sta-article-3-3/24132
+        // ---------- Approve Coinbase Custody (RWA014-A) Dao Resolution ----------
+        // Forum: https://forum.makerdao.com/t/proposal-to-extend-coinbase-custody-rewards/24198/4
 
-        // Trigger Spark Proxy Spell at 0x151D5fA7B3eD50098fFfDd61DB29cB928aE04C0e
-        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
+        // Approve Coinbase Custody (RWA014-A) Dao Resolution with IPFS hash QmbyXH3z74wQqTDPNFKseLbVAJwKmHwp23AQZPppNZfkaD
```

Comment:
```suggestion
        // Approve Coinbase Custody (RWA014-A) Dao Resolution with IPFS hash QmUvqDK4EgGFw6dUqmdhcoZiSBLFuhf2e6k63WwrauRUx9
```
Dao resolution doens't match with exec sheet

Reply:
Yeah, looks like they changed it. I was about to go over it again. Thanks for flagging.